hw/dv/data/common_sim_cfg.hjson

```diff
@@ -24,7 +24,7 @@
   // pass and fail patterns
   build_pass_patterns: []
   build_fail_patterns: ["^ERROR:.*$"] // fusesoc build error
-  run_pass_patterns:   ["^TEST PASSED (UVM_)?CHECKS$"]
+  run_pass_patterns:   ["^(# KERNEL: )?TEST PASSED (UVM_)?CHECKS$"]
```

Comment:
Our flow prints this identifier on a new line as a `UVM_INFO`. This should not be needed.